src/larvaworld/lib/reg/models.py

```diff
@@ -929,6 +935,7 @@ def autogenerate_confs(self):
         E['max_feeder'] = self.newConf(m0=E['RE_NEU_PHI_DEF_max_feeder'], kwargs={})
         E['explorer'] = self.newConf(m0=E['loco_default'], kwargs={})
         E['navigator'] = self.newConf(m0=E['explorer'], kwargs=kwargs1)
+        E['navigator_osn'] = self.newConf(m0=E['explorer'], mID='navigator_osn', kwargs=olf_brian_osn_kwargs)
```

Comment:
what is does 'explorer' here mean ?

The function and parameter names are a bit uninformative 🐙